### DISCUSSION TITLE
Capturing screen saver on Windows 11?

### DISCUSSION THREAD

**Discussion by darkj2k** [obsproject/obs-studio#6177] [General]
I'm tried to capturing screen saver on Windows 11. But the result video is black screen when screen saver start. So is it possible to recording the screen saver? I'm trying to capture the Mystify screen saver as a video. Thanks in advance.

**Comment by henke37** [comment]
Are you running the screen saver on the secure desktop? Generally if you have a log in password determinates if screen savers run on the secure desktop or the normal desktop.

**Reply by darkj2k** [reply]
I have "On resume, display logon screen" disabled.

**Comment by Fenrirthviti** [comment]
This sounds more likely to be a support request, and we are not currently accepting support requests on GitHub Discussions. Make sure you read any [pinned posts](https://github.com/obsproject/obs-studio/discussions/5206) on discussion boards before opening a new discussion. Please use our [forums](https://obsproject.com/forum/) or [Discord](https://obsproject.com/discord/) for further assistance.

Thank you!
